src/firebase-import.js

```diff
@@ -80,29 +82,74 @@ function promptToContinue(ref, next) {
   }
 }
 
+function getJsonFromFile(file, callback) {
+  var firstChar;
+  var rs = fs.createReadStream(file);
+  rs.on('data', function(chunk) {
+    var s = chunk.toString().trim();
+    if (s !== "") {
+      rs.close();
+    }
+    firstChar = s[0];
+  })
+  .on('close', function() {
+    if (firstChar === "[" || firstChar === "{") {
+      var jsonStream;
+      var onFunc;
+      var json;
+      if (firstChar === "[") {
+        console.log("Reading json as an array...");
+        json = [];
+        jsonStream = JSONStream.parse("*");
+        onFunc = function(r) {
+          json.push(r);
+        };
+      } else {
+        console.log("Reading json as an object...");
+        json = {};
+        jsonStream = JSONStream.parse("$*");
+        onFunc = function(r) {
+          json[r.key] = r.value;
+        };
+      }
+      fs.createReadStream(file)
+      .pipe(jsonStream)
```

Comment:
Do you mind indenting these lines (probably 4 spaces) since they're continuations of the previous line?